tests/UKHO.S100PermitService.Common.UnitTests/Services/PermitServiceTests.cs

```diff
@@ -136,10 +140,13 @@ public async Task WhenEmptyPermitXml_ThenFileIsNotCreated()
         {
```

Comment:
Similarly in this test case, add spacing between to fake calls

Reply:
Done